### PR TITLE
create resource group/service plan from webapp creation dialog

### PR DESCRIPTION
**Pull request by wangmingliang-ms**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/resourcegroup/ResourceGroupCreationDialog.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        return builder.build();
+    }
+
+    @Override
+    public List<AzureFormInput<?>> getInputs() {
+        return Collections.singletonList(this.textName);
+    }
+
+    private void createUIComponents() {
+        this.labelDescription = new JLabel("<html><body><p>" + DESCRIPTION + "</p></body></html");
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/resourcegroup/ResourceGroupCreationDialog.java:97]: miss '>' in `"</p></body></html"`?

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/resourcegroup/ResourceGroupCreationDialog.java:97]: why we need to use html here? for wrap new line?

Comment (wangmingliang-ms) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/resourcegroup/ResourceGroupCreationDialog.java]: @Flanker32  yes.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/appservice/ResourceGroupMock.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import rx.Observable;
+
+import java.util.Map;
+
+@Setter
+@Builder
+public class ResourceGroupMock implements ResourceGroup, Mock {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/appservice/ResourceGroupMock.java:43]: ResourceGroupMock seems a class only for test, a bit confusing

Comment (andxu) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/appservice/ResourceGroupMock.java:43]: Consider to change it to PlaceHolder

**Review by Flanker32 — APPROVED** [review]

**Pull request merged**